src/styles/blocks/header.scss

```diff
@@ -0,0 +1,65 @@
+.header {
+  @extend %flex-between;
+  background-attachment: $prime-color;
+  padding: 0 50px;
+  height: 60px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  &__logo {
+    margin: 10px 0;
+  }
+}
+
+.nav {
+  text-align: center;
+  font-weight: 500;
+
+  &__link {
+    display: inline-block;
+    position: relative;
+    text-decoration: none;
+    padding: 23px 0;
+    text-transform: uppercase;
+    color: #000;
+    font-size: 12px;
+
+    &:hover {
+      color: $scnd-color;
+      transition: 400ms;
```

Comment:
transition should be included in the &__link selector, not in hover. Also please include for which rule you want to apply transition (e.g.color or transform).